src/o365/spo/commands/get.ts

```diff
@@ -0,0 +1,71 @@
+import commands from '../commands';
+import GlobalOptions from '../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandValidate,
+  CommandError
+} from '../../../Command';
+import SpoCommand from '../../base/SpoCommand';
+import auth from '../../../Auth';
+import { SpoContext } from './SpoContext';
+
+const vorpal: Vorpal = require('../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {}
+
+class SpoGetCommand extends SpoCommand {
+  public get name(): string {
+    return `${commands.GET}`;
+  }
+
+  public get description(): string {
+    return 'Gets the context URL for the root SharePoint site collection and SharePoint tenant admin site.';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    if (!auth.service.connected) {
+      cb(new CommandError('Login to Office 365 first'));
+      return;
+    }
+
+    const spoContext: SpoContext = {
+      SpoUrl: auth.service.spoUrl ? auth.service.spoUrl : ''
+    };
+    cmd.log(spoContext);
+    cb();
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+      return true;
+    };
+  }
+
+  public commandHelp(args: any, log: (help: string) => void): void {
+    log(vorpal.find(commands.GET).helpInformation());
+    log(` Remarks:
+
+    Office 365 CLI automatically discovers the URL of the root SharePoint site
+    collection/SharePoint tenant admin site (whichever is needed to run
+    the particular command). Using this command you can see which URLs the CLI has discovered.
+      
+  Examples:
+  
+      Get the context URL for the root SharePoint site collection and SharePoint tenant admin site.
+      ${commands.GET} -o json
```

Comment:
Changed to use the full option name e.g. `${commands.GET} --output json` instead of just `-o json`